lib/class-wp-rest-sidebars-controller.php

```diff
@@ -370,6 +371,10 @@ public function prepare_item_for_response( $raw_sidebar, $request ) {
 			$sidebar['status'] = 'inactive';
 		}
 
+		if ( 'wp_inactive_widgets' === $sidebar['id'] && empty( $sidebar['name'] ) ) {
```

Comment:
Maybe make `'wp_inactive_widgets' === $sidebar['id'] && empty( $sidebar['name'] )` into another `is` variable `$is_inactive_widgets_sidebar` just to make it like `$is_registered_sidebar` ... not important.

Reply:
It would have to be just `$is_inactive_widgets_sidebar = 'wp_inactive_widgets' === $sidebar['id']`, the name may or may not be empty. It would be a tad more readable, but since that line appears only once per method and there is no duplication, the benefits are arguable - I'd say it's a matter of taste.